app/helpers/application_helper.rb

```diff
@@ -64,4 +64,8 @@ def using_non_primary_url?
   def survey_link
     t("service.public_beta_survey_link")
   end
+
+  def external_cw_form_link
```

Comment:
As this helper is a one-liner that's only used in one place, I think you could reasonably just move the content of this method into the controller. (If you're wondering how to access the `t` helper within a controller, the way we typically do it is by calling `I18n.t`, but now I wonder if `helpers.t` would also work)